### PR TITLE
fix start.bat support agent parameter[shenyu-bootstrap-dist missing agent parameter in start.bat script (Issue #2814)]

### PR DESCRIPTION
**Pull request by IceFoxs**
fix start.bat support agent parameter Issue #2814

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-dist/shenyu-bootstrap-dist/src/main/resources/bin/start.bat [hunk 1]
```diff
@@ -1,46 +1,58 @@
...
+@rem the License.  You may obtain a copy of the License at
+@rem
+@rem     http://www.apache.org/licenses/LICENSE-2.0
+@rem
+@rem Unless required by applicable law or agreed to in writing, software
+@rem distributed under the License is distributed on an "AS IS" BASIS,
+@rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@rem See the License for the specific language governing permissions and
+@rem limitations under the License.
+@rem
```
Comment (yu199195) [inline shenyu-dist/shenyu-bootstrap-dist/src/main/resources/bin/start.bat:16]: why not modify all file？

File: shenyu-dist/shenyu-bootstrap-dist/src/main/resources/bin/start.bat [hunk 2]
```diff
@@ -1,46 +1,58 @@
...
+set BASE_DIR=%~dp0
+set BASE_DIR=%BASE_DIR:~0,-5%
+
+set LOG_HOME=%BASE_DIR%/logs
+set SERVER_NAME=ShenYu-Bootstrap
+
+set CLASS_PATH=".;..\conf;..\lib\*;..\ext-lib\*"
+
+set JAVA_OPTS=-server -Xmx2g -Xms2g -Xmn1g -Xss256k -XX:+DisableExplicitGC  -XX:LargePageSizeInBytes=128m
+for /f tokens^=2-5^ delims^=^" %%j in ('java -fullversion 2^>^&1') do set "version=%%j"
```
Comment (yu199195) [inline shenyu-dist/shenyu-bootstrap-dist/src/main/resources/bin/start.bat:30]: wants mean

File: shenyu-dist/shenyu-bootstrap-dist/src/main/resources/bin/start.bat (hunk 1 above)
Comment (IceFoxs) [reply shenyu-dist/shenyu-bootstrap-dist/src/main/resources/bin/start.bat:16]: I edit this file in windows，copy to other pc submit file，git identify all file midified.I resubmit it.

**Review by IceFoxs — COMMENTED** [review]
I remodify it, you can check it again!

File: shenyu-dist/shenyu-bootstrap-dist/src/main/resources/bin/start.bat (hunk 2 above)
Comment (IceFoxs) [reply shenyu-dist/shenyu-bootstrap-dist/src/main/resources/bin/start.bat]: > wants mean

Sorry, I already  remodify it.
Can you review  it?

**Review by yu199195 — APPROVED** [review]

**Pull request merged**